articles/app-service/app-service-hybrid-connections.md

```diff
@@ -22,160 +22,165 @@ ms.author: ccompy
 - Active Directory support
 
 ## Adding and Creating a Hybrid Connection in your App ##
 
-Hybrid Connections can be created through the app portal or from the Hybrid Connection service portal.  It is highly recommended that you use the app portal to create the Hybrid Connections you wish to use with your app.  To create a Hybrid Connection go to the [Azure portal][portal] and go into the UI for your app.  Select **Networking > Configure your hybrid connection endpoints**.  From here you can see the Hybrid Connections that are configured with your app  
+Hybrid Connections can be created through your App Service app in the Azure portal or from Azure Relay in the Azure portal.  It is highly recommended that you create Hybrid Connections through the App Service app that you wish to use with the Hybrid Connection.  To create a Hybrid Connection, go to the [Azure portal][portal] and select your app.  Select **Networking > Configure your Hybrid Connection endpoints**.  From here you can see the Hybrid Connections that are configured for your app.  
 
 ![][2]
 
-To add a new Hybrid Connection, click Add Hybrid Connection.  The UI that opens up lists the Hybrid Connections that you have already created.  To add one or more of them to your app, click on the ones you want and hit **Add selected hybrid connection**.  
+To add a new Hybrid Connection, click Add Hybrid Connection.  The UI that opens up lists the Hybrid Connections that you have already created.  To add one or more of them to your app, click on the ones you want and hit **Add selected Hybrid Connection**.  
 
 ![][3]
 
-If you want to create a new Hybrid Connection, click **Create new hybrid connection**.  From here you specify the: 
+If you want to create a new Hybrid Connection, click **Create new Hybrid Connection**.  From here, you specify the: 
 
-- endpoint name
-- endpoint hostname
-- endpoint port
-- servicebus namespace you wish to use
+- Endpoint name
+- Endpoint hostname
+- Endpoint port
+- Service Bus namespace you wish to use
 
 ![][4]
 
-Every hybrid connection is tied to a service bus namespace and each service bus namespace is in an Azure region.  It is important to try and use a service bus namespace in the same region as your app so as to avoid network induced latency.
+Every Hybrid Connection is tied to a Service Bus namespace, and each Service Bus namespace is in an Azure region.  It is important to try and use a Service Bus namespace in the same region as your app so as to avoid network induced latency.
 
-If you want to remove your hybrid connection from your app, right click on it and select **Disconnect**.  
+If you want to remove your Hybrid Connection from your app, right click on it and select **Disconnect**.  
 
-Once a hybrid connection is added to your web app, you can see details on it by simply clicking on it.  
+Once a Hybrid Connection is added to your app, you can see details on it by simply clicking on it.  
 
 ![][5]
 
```

Comment:
Please insert the following item here.  

### Creating a Hybrid Connection in the Azure Relay portal ###

In addition to the portal experience from within your app, there is also an ability to create Hybrid Connections from within the Azure Relay portal.  In order for a Hybrid Connection to be used by the Azure App Service it must satisfy two criteria.  It must:

* Require Client Authorization
* Have a metadata item named **endpoint** that contains a **<host:port>** combination as the value.